Sources/Fuzzilli/Compiler/JavaScriptParser.swift

```diff
@@ -57,7 +57,7 @@ public class JavaScriptParser {
     private func runParserScript(withArguments arguments: [String]) throws {
         let output = Pipe()
         let task = Process()
-        task.standardOutput = output
+        // task.standardOutput = output // ignore stdout to avoid deadlock caused by full Pipe()
```

Comment:
Could you just make this a comment and delete the code instead, e.g.
```
// Don't set standardOutput: we only need stderr for error reporting and
// capturing stdout here may cause a deadlock if the pipe becomes full.
```

Reply:
of coures